emitter.js

```diff
@@ -54,6 +54,7 @@ class EventEmitter extends events.EventEmitter {
     let emitter = this;
     let promise;
 
+    this._event = type;
```

Comment:
This won't work as when the `emit` method is called twice, the property will be set with the last event type, making the value unpredictable or undefined. What is the goal for this anyway?